### ISSUE TITLE
Add first PR condition to GitHub maintainer workflow

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#300]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

One thing that would make the maintainer experience a lot better for Scribe-Data and other projects would be if the [.github/workflows/pr_maintainer_checklist.yaml](https://github.com/scribe-org/Scribe-Data/blob/main/.github/workflows/pr_maintainer_checklist.yaml) had a conditional for if it's the first pull request to the repository. If it is the first pull request, then we would want the first checkbox to be included, and if not, then the first checkbox should be removed.

In short: The box is very useful for new contributors, but I'd like to stop checking it for people who I know have their account set up properly 😅 

### Contribution

Would be really happy to review this and see it working!

**Comment by axif0** [comment]
Can we use GitHub Actions expressions to check the pull request number? If it's the first pull request (github.event.pull_request.number == 1), we can include the specific checklist item.


```
${{ if eq(github.event.pull_request.number, 1) }}
- [ ] The commit messages for the remote branch should be checked to make sure the contributor's email is set up correctly so that they receive credit for their contribution
 - The contributor's name and icon in remote commits should be the same as what appears in the PR
- If there's a mismatch, the contributor needs to make sure that the [email they use for GitHub](https://github.com/settings/emails) matches what they have for `git config user.email` in their local Scribe-Data repo
${{ endif }}
```


- [ref](https://github.com/actions/checkout/issues/58#issuecomment-1190882933)

**Comment by andrewtavis** [comment]
I think we're looking for more leveraging [actions/first-interaction](https://github.com/actions/first-interaction) here, @axif0 :) What you suggested is more for the first pull request on the repo, not for the first pull request of a user on the repo 😊

**Comment by axif0** [comment]
@andrewtavis  Want to work on this. Can I assign here if there is no problem?

**Comment by andrewtavis** [comment]
Closed by #308 😊 Thanks a lot, @axif0!

**Comment by andrewtavis** [comment]
Thanks for all of these, @axif0! 😊
